build.sbt

```diff
@@ -86,6 +86,10 @@ val algebraVersion = "2.9.0"
 val annoy4sVersion = "0.10.0"
 val annoyVersion = "0.2.6"
 val breezeVersion = "2.1.0"
+val bsonVersion = "4.8.1"
+val cosmosVersion = "4.37.1"
+val cosmosContainerVersion = "1.17.5"
+val scribeVersion = "3.10.7"
```

Comment:
The production code is implement with Slf4j (`import org.slf4j.LoggerFactory`)

![imagen](https://user-images.githubusercontent.com/9974/216072022-737a6b2f-40b5-4579-a824-b277280e0965.png)

But the dependency is implicit, I'm add the dependency explicit.

And respect with scribe, I'm using only in test, because is gives me explicit number line without overload.

Reply:
Yes. It's more a coherence issue. As maintainer, it is preferable to have all module look as similar as possible.
It's also the same with the beam implementations. We've taken the habit to write them in java so upstream contribution to beam can be easy.
Since you've also opened a PR in beam, would you like some support on that ?

Reply:
Is there no possibility to support scribe only in tests? It's very scala style, plus it looks beautiful :)

![imagen](https://user-images.githubusercontent.com/9974/216109836-c36daa89-182a-491b-abf9-3e6b631d3309.png)